packages/create-react-native-library/templates/native-common/android/build.gradle

```
@@ -18,6 +18,11 @@ buildscript {
   }
 }
 
+def reactNativeArchitectures() {
+  def value = project.getProperties().get("reactNativeArchitectures")
```

Shouldn't this be `rootProject`?

```suggestion
  def value = rootProject.getProperties().get("reactNativeArchitectures")
```